js-miniapp-sample/src/pages/user-details.js

```diff
@@ -310,11 +310,27 @@ function UserDetails(props: UserDetailsProps) {
           )}
           {props.contactList &&
             props.contactList.map((contact) => (
-              <ListItem>
+              <ListItem divider>
                 <ListItemAvatar>
                   <Avatar className={classes.contactIcon} />
                 </ListItemAvatar>
-                <ListItemText primary={contact.id} />
+                <ListItemText
+                  primary={contact.id}
+                  secondary={
+                    <React.Fragment>
+                      <Typography>
+                        {contact.name !== '' ? (
```

Comment:
Will this be passed as an empty string when it doesn't exist? 

But `name` and `email` are optional properties, so I think they should be passed as `null` or `undefined` instead of an empty string.

Reply:
That's right. From iOS SDK, it is optional and it will pass as an empty string by default unless the Host app assigns `nil` to `name` and `email`.

Now, updated the code to not display the `name` & `email` if the value is `undefined` or `null`